Documentation/README.md

```diff
@@ -1,42 +1,43 @@
 # Ladybird Documentation
 
-Serenity development moves quickly, so some of these might be out of date. Please let us know if something here is wrong, or submit a PR with any additions or corrections! If you have any questions that are not answered here or in the [FAQ](FAQ.md), you are welcome to ask on [Discord](../README.md#get-in-touch-and-participate).
+Ladybird development moves quickly, so some of these might be out of date.
+Please let us know if something here is wrong, or submit a PR with any
+additions or corrections! If you have any questions that are not answered here
+or in the [FAQ](FAQ.md), you are welcome to ask on
+[Discord](../README.md#get-in-touch-and-participate).
```

Comment:
Line breaking at 70ish characters is a bit much IMO - let's keep it on one line like it was before, or break less frequently (120 chars is the soft limit we use in other places)

Reply:
Also here, vim line break config at fault, will change.